agent/06_agent_create_cluster.sh

```diff
@@ -243,6 +303,21 @@ if [ ! -z "${AGENT_ENABLE_GUI:-}" ]; then
   enable_assisted_service_ui
 fi
 
+if [[ "${AGENT_USE_APPLIANCE_MODEL}" == true ]] && [[ "${AGENT_APPLIANCE_HOTPLUG}" == true ]]; then
```

Comment:
Is there any difference between having the config image already attached to the host when the host first boots, versus mounting it say 2 minutes after the host has booted?

Reply:
Functionally there is no difference, the contents will be copied when the config-image can be read. This flag is mainly useful to allow checking the contents of the unconfigured ignition prior to copying the config-image. I don't expect this flag will be used with automated testing, but its useful for demos and manual testing.